engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -346,7 +346,19 @@ class WorkflowActor(val workflowId: WorkflowId,
   }
 
   private[workflow] def makeFinalizationActor(workflowDescriptor: EngineWorkflowDescriptor, jobExecutionMap: JobExecutionMap, workflowOutputs: CallOutputs) = {
-    context.actorOf(WorkflowFinalizationActor.props(workflowId, workflowDescriptor, ioActor, jobExecutionMap, workflowOutputs, stateData.initializationData), name = s"WorkflowFinalizationActor")
+    val copyWorkflowOutputsActorProps = stateName match {
+      case InitializingWorkflowState => None
+      case _ => Option(CopyWorkflowOutputsActor.props(workflowIdForLogging, ioActor, workflowDescriptor, workflowOutputs, stateData.initializationData))
+    }
+    
+    context.actorOf(WorkflowFinalizationActor.props(workflowId,
+      workflowDescriptor,
+      ioActor,
+      jobExecutionMap,
+      workflowOutputs,
+      stateData.initializationData,
+      copyWorkflowOutputsActorProps
+    ), name = s"WorkflowFinalizationActor")
```

Comment:
Woohoo!
Bonus points if you also explicitly name the inputs so that there's no chance of untyped `ActorRef`s getting accidentally reordered